frontend/src/pages/faq.page.tsx

```diff
@@ -6,154 +6,163 @@ import { Layout } from "../components/layout/Layout";
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-limit")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-call-length"}
-        question={l10n.getString("phone-masking-faq-question-call-length")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-call-length")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-can-i-call"}
-        question={l10n.getString("phone-masking-faq-question-can-i-call")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-can-i-call")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-can-i-see"}
-        question={l10n.getString("phone-masking-faq-question-can-i-see")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-can-i-see")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-can-i-block"}
-        question={l10n.getString("phone-masking-faq-question-can-i-block")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-can-i-block")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-spam"}
-        question={l10n.getString("phone-masking-faq-question-spam")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-spam")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-disable-logging"}
-        question={l10n.getString("phone-masking-faq-question-disable-logging")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-disable-logging")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-can-i-share"}
-        question={l10n.getString("phone-masking-faq-question-can-i-share")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-can-i-share")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-how-i-save-card"}
-        question={l10n.getString("phone-masking-faq-question-how-i-save-card")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-how-i-save-card")}</p>
-      </QAndA>
-      <QAndA
-        id={"phone-masking-faq-question-install-app"}
-        question={l10n.getString("phone-masking-faq-question-install-app")}
-      >
-        <p>{l10n.getString("phone-masking-faq-answer-install-app")}</p>
-      </QAndA>
-      <QAndA
-        id="phone-masking-faq-question-data"
-        question={l10n.getString("phone-masking-faq-question-data")}
-      >
-        <Localized
-          id="phone-masking-faq-answer-data"
-          vars={{
-            url: "https://www.mozilla.org/privacy/firefox-relay/",
-            attrs: "",
-          }}
-          elems={{
-            a: (
-              <a
-                href="https://www.mozilla.org/privacy/firefox-relay/"
-                target="_blank"
-                rel="noopener noreferrer"
-              />
-            ),
-          }}
+  const phoneMaskingFaqs =
+    isPhonesAvailableInCountry(runtimeData.data) &&
```

Comment:
Literally the only change here is the addition of this line, but reformatting caused indenting changes causing all other changes in this file :sweat_smile: